serialization/json-ld.md

```diff
@@ -0,0 +1,41 @@
+# JSON-LD
+JSON-LD is a JSON-based format to encode RDF graphs. Its documentation can be found [here](https://www.w3.org/TR/json-ld11/).
+
+The JSON-LD is an RDF format and follows the serialization rules of the [rdf SPDX serialization specification](rdf.md).
+
+## XCollection
+
+The following XCollection properties are mapped to JSON-LD syntax specifications.
+Any properties not listed below should be serialized as part of the XCollection element itself within
+the JSON-LD serialized data.
+
```

Comment:
Needs new additional line(s)after line 10:

"Deserialization of any JSON-LD serialized SPDX content MUST expand the inverse of these native mappings such that the logical "XCollection" element directly contains its full set of properties."

Reply:
Updated per suggestion